example/workflows/browseExample.yml

```diff
@@ -162,6 +162,7 @@ CookbookBrowseExample:
         bucket: '{$.meta.buckets.internal.name}'
         buckets: '{$.meta.buckets}'
         distribution_endpoint: '{$.meta.distribution_endpoint}'
+        cmr: '{$.meta.cmr}'
```

Comment:
So I think instead of this what we should do is pass CMR_ENVIRONMENT into all our lambdas as an environment variable (this can be done in the cloudformation template where we generate the lambdas, wherever we create `AWS::Lambda::Function`). That way the users don't miss a config somewhere, since it would be really bad if they do.

Reply:
Changed 👍